### ISSUE TITLE
Image Recognition using Quantum Machine Learning

### ISSUE THREAD

**Issue by YumaNK** [qiskit-community/qiskit-hackathon-taiwan-20#24]
# Abstract
Using Quantum Machine Learning (such as Quantum Support Vector Machine; QSVM), classify images of two types of digits (0 or 1). Since current QSVM just able to process few feature dimension, one need to classically convert image data into data with few dimension (2~4dim) using PCA(ML technique to reduce dimension.) After its process, one can use quantum machine learning to classify its label.
If the project reaches certain level,  the output might deserve the tutorial in [Qiskit Community](https://github.com/Qiskit/qiskit-community-tutorials)

![image](https://user-images.githubusercontent.com/19603134/92436740-5c327380-f1e0-11ea-89a0-1368bb58bbd7.png)

# Description 
Useful reference:
   ・Tutorial of QSVM [link](https://github.com/qiskit-community/qiskit-community-tutorials/blob/master/machine_learning/qsvm.ipynb)
   ・Image dataset used in Quantum Neural Network [link](https://qiskit.org/textbook/ch-machine-learning/machine-learning-qiskit-pytorch.html)
# Members

 - @githubhandle 
 - @@githubhandle - Slack: @slackhandle email: example@example.com
 - Qiskit Coach: @YumaNK 

# Deliverable

# GitHub repo

**Comment by treger-8105** [comment]
Hi I want to join this project.
Can I join ?
